pkgs/default.nix

```diff
@@ -1,7 +1,19 @@
 {
   lib,
   callPackage,
+  inputs,
 }: let
+  linuxSystem = "x86_64-linux";
```

Comment:
@lorenzleutgeb I'm not sure what to do with this. I'm quite sure I don't understand the overall structure of the flake and that's why I'm confused. I know it's documented with some comments, but I still don't get it. Perhaps I'll take another look. There's just too much passing of values around and I lose track.